app/views/transition_landing_page/_countdown.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="Transition-countdown govuk-grid-row">
+  <div class="govuk-grid-column-one-half Transition-countdown__countdown">
+    <p class="Transition-countdown__countdown-days-left"><%= clock.days_left %></p>
+    <p class="Transition-countdown__countdown-days-text"><%= clock.days_text %> to go</p>
```

Comment:
Worried that splitting (for example) `92 days to go` up with `p` tags is an accessibility fail, as screen readers will read it out with a big pause in the middle.

Reply:
Should we split using a <br> tag instead?

Reply:
roger